src/utils/constants.cxx

```diff
@@ -152,3 +152,15 @@ DEFAULT_CONST(gridconnect_bridge_max_incoming_packets, 1);
 DEFAULT_CONST(gridconnect_bridge_max_outgoing_packets, 1);
 
 DEFAULT_CONST_FALSE(gridconnect_tcp_use_select);
+
+#ifdef ESP32
+/// Use a stack size of 3kb for SocketListener tasks.
+DEFAULT_CONST(socket_listener_stack_size, 3072);
```

Comment:
this used to be 2k, did it need to be bumped to 3k?

Reply:
Yes, it was unfortunately needed after further testing as part of ESP32 CS.